src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,14 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  categories: [],
+  status: 'Under construction',
+};
+
+const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState,
+  reducers: {},
+});
+
+export const { action: categoriesActions, reducer: categoriesReducer } = categoriesSlice;
```

Comment:
- [ ] Kindly export your reducer and action as required. You are exporting action and reducer that are not present in this file. After adding the reducer above, kindly export the reducer and action like this below:

```javascript
export const { checkStatus } = categoriesSlice.actions

export default categoriesSlice.reducer
```

Reply:
Correction done

Reply:
Thank you @AmaduKamara